docs/template_usage.rst

```diff
@@ -176,13 +176,16 @@ Other ipywidgets can be embedded by setting them in a trait and adding widget_se
             'content': slider2
         }]).tag(sync=True, **widgets.widget_serialization)
 
-        single_widget = traitlets.Any(
-            widgets.IntSlider(description='Single slider', value=40)
-        ).tag(sync=True, **widgets.widget_serialization)
+        single_widget = traitlets.Any().tag(sync=True, **widgets.widget_serialization)
+    
+        def __init__(self, **kwargs):
+            self.single_widget = widgets.IntSlider(description='Single slider', value=40)
+            self.single_widget.__init__(**kwargs)
```

Comment:
Calling the constructor twice cannot be right, it probably has weird side effects (creating 2 comms).
```suggestion
            self.single_widget = widgets.IntSlider(description='Single slider', value=40, **kwargs)
```

Reply:
I agree, but the suggested change does not work when calling MyComponent(value=20), i.e. for arguments already defined.
```suggestion
            self.single_widget = widgets.IntSlider(description='Single slider', value=40)
            for k, v in kwargs.items():
                setattr(self.single_widget, k, v)
```
Alternatively, the default arguments are passed to `__init__`:
```
def __init__(self, description='Single slider', value=40, **kwargs):
    self.single_widget = widgets.IntSlider(
        description=description, 
        value=value, 
        **kwargs
    )
```
The second option may look more readable. Any preference?